borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+will create a frameless window removing all UA provided Chrome, leaving only
+resizing of the window from its borders. The removed window controls will be
+enabled using AWC and HTML/JavaScript/CSS (see non-goals and
+[additional-windowing-controls explainer](<[http://go/additional-windowing-controls](https://github.com/ivansandrk/additional-windowing-controls/blob/main/awc-explainer.md)>)).
+Additionally to hiding the native title bar, also the menu button is hidden.
+
+Example apps in borderless mode could look e.g. like below but the appearance
+would eventually fully depend on what the developer would implement.
+
+![Example borderless PWA](./images/borderless-app-with-AWC.png)
+
+The desire to remove the title bar will be declared within the web app manifest
+by setting the `display_override` to `borderless`. The display_override value
+will be ignored on unsupported OSs and they will follow the normal display mode
+hierarchy.
+
+```
+{
+  "display_override": [ "borderless" ]
+}
+```
+
+### 2. Enabling draggable regions
+
+When the app enters the borderless mode,
+[draggable regions](https://github.com/WICG/window-controls-overlay/blob/main/explainer.md#defining-draggable-regions-in-web-content)
+`-webkit-app-region: drag;` will be enabled.
+
+### 3. CSS display-mode media query for borderless
+
+Similarly to other display modes, borderless should be queryable with @media.
+
+Example media query with borderless:
+
+```
+@media (display-mode: borderless) {
+  .any-css-class-name {
+    margin: 5px;
+  }
+}
+```
+
+See:
+[https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode)
+
+### 4. Leveraging Window Management permission for borderless
+
+In order for the borderless mode to activate, the app needs to have Window
+Management permission granted. The permission prompt can be triggered using
+JavaScript.
+
+For the **managed** context, permission prompts can be by-passed with a policy
+set by an admin and later on edited in the app’s settings. For the **unmanaged**
+context, the Window management API permission must be granted.
+
+#### Enabling Window Management
+
+Granting the window management API permission can be done with the following
+JavaScript:
+
+```
+const screenDetails = await window.getScreenDetails();
+```
+
+That will show the permission prompt to enable Window Management API permission
+which will also enable borderless capability in case the permission is given.
+
+Later on, the state of the permission can be queried with the following script:
+
+```
+navigator.permissions.query({name:'window-placement'})
+  .then((status) => {
+    // Do what you need with the permission state.
+    console.log(status.state)
+  });
+```
+
+#### Disabling Window Management
+
+To disable borderless mode, the user can disable the Window Management
+permission in App Settings.
+
+In case the Window management permission is not provided or no longer in place,
```

Comment:
Ah I see this answers my above question. I would rearrange this so you explain the flow up front: since in most cases you will start off with the permission not being provided, it would be good to explain first what the user sees when the permission has not been granted, and then how they get into this mode.